### PR TITLE
Feat/aave v2 withdrawals

### PR DESCRIPTION
**Pull request by dimlbc**
https://trello.com/c/nylMpK9G/4736-implement-new-aave-withdrawal-flow-in-trading-widget

New withdrawals flow will be used for selected networks determined by `aaveOffchainWithdrawChainIds`.
Withdrawals on other networks will works without any changes.

TODO:  
- [x] replace DEV_FACTORY and DEV_EASY_SWAPPER with prod addresses before release
- [x] configure `aaveOffchainWithdrawChainIds`
- [x] bump widget version to 3.3.0

### CODE REVIEW

File: packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-fetch-init-withdraw-complex-asset-data.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+              return buildAaveWithdrawAssetTransactionData({
+                assetAddress: asset,
+                swapData,
+                swapParams,
+                slippage,
+              })
+            } catch (error) {
+              console.error(error)
+
+              return {
```
Comment (artsiomYavorski) [inline packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-fetch-init-withdraw-complex-asset-data.ts:65]: looks like duplication of returned value in catch block and outside of if block. Could be removed here?
or `slippageTolerance` value will be different as @D-Ig suggests

File: packages/trading-widget/src/core-kit/hooks/pool/multicall/use-pool-manager.dynamic.ts [hunk 2]
```diff
@@ -32,42 +20,35 @@ const getContracts = ({
...
-      address,
+      address: managerLogicAddress ?? AddressZero,
       chainId,
-      managerLogicAddress: managerLogicAddress ?? AddressZero,
-      account,
     }),
     query: {
       enabled: !!managerLogicAddress && !isZeroAddress(managerLogicAddress),
       select: selector,
+      refetchInterval: DEFAULT_POLLING_INTERVAL,
```
[the remark this answers is not in the document]
Comment (dimlbc) [reply packages/trading-widget/src/core-kit/hooks/pool/multicall/use-pool-manager.dynamic.ts:51]: This is only for cases when the browser tab remains open for a long period of time. There is no need to decrease this interval, as everything will be re-fetched after each transaction.

File: packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-fetch-init-withdraw-complex-asset-data.ts (hunk 1 above)
Comment (dimlbc) [reply packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-fetch-init-withdraw-complex-asset-data.ts:68]: `slippageTolerance` will be different

File: packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-fetch-init-withdraw-complex-asset-data.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    }): Promise<ComplexWithdrawAssetData[]> => {
+      if (disabled) {
+        return []
+      }
+      const isOffchainAaveWithdrawSupported = new BigNumber(
+        withdrawAmount.toString(),
+      )
+        .shiftedBy(-DEFAULT_PRECISION)
+        .times(vaultTokenPrice)
+        .gt(aaveOffchainWithdrawMinValue)
```
Comment (D-Ig) [inline packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-fetch-init-withdraw-complex-asset-data.ts:53]: what decimals has
* `withdrawAmount`?
* `vaultTokenPrice`?
* `aaveOffchainWithdrawMinValue`?

File: packages/trading-widget/src/core-kit/hooks/pool/use-has-dhedge-vault-in-composition.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  chainId,
+}: PoolContractCallParams) => {
+  const composition = usePoolComposition({
+    address,
+    chainId,
+  })
+
+  return useReadContracts({
+    contracts: composition.map(({ tokenAddress }) => ({
+      address: getContractAddressById('factory', chainId),
```
Comment (D-Ig) [inline packages/trading-widget/src/core-kit/hooks/pool/use-has-dhedge-vault-in-composition.ts:18]:
```suggestion
  const poolFactoryAddress = getContractAddressById('factory', chainId)
  return useReadContracts({
    contracts: composition.map(({ tokenAddress }) => ({
      address: poolFactoryAddress,
```

**Review by D-Ig — APPROVED** [review]

**Pull request merged**